chromium_src/chrome/browser/resources/tab_search/auto_tab_groups/auto_tab_groups_page.ts

```diff
@@ -143,6 +146,9 @@ export class AutoTabGroupsPageElement extends CrLitElement {
   }
 
   override render() {
+    if (this.showFRE_) {
```

Comment:
niiiiice. Lit is cool.

Optionally: We could do this in the `.html.ts` file? As a bonus I think we could skip doing the `.bind(this)()` as the `this` context would be implied

Reply:
Moved the return check to the `.html.ts` file.